coretex/utils/number.py

```diff
@@ -39,3 +39,25 @@ def mathematicalRound(value: float, decimalPlaces: int) -> float:
     places = Decimal(10) ** -decimalPlaces
 
     return float(decimal.quantize(places, rounding = ROUND_HALF_UP))
+
+
+def formatBytes(value: int, precision = 2):
+    """
+    Format a size in bytes into a human-readable format.
+
+    Parameters:
+        value (int): The size in bytes.
+        precision (int): Number of decimal places (default is 2).
+
+    Returns:
+        str: Formatted string with size and appropriate unit.
```

Comment:
Use the same formatting as we do for documentation of other functions

Reply:
Updated